webapp/components/interop.js

```diff
@@ -406,36 +406,41 @@ class InteropDashboard extends PolymerElement {
 
         .table-card {
           height: 100%;
-          display: flex;
-          border-radius: 3px;
-          background: white;
         }
 
         .score-table {
-          height: 100%;
           width: 100%;
           border-collapse: collapse;
+          margin-top: 3em;
```

Comment:
```suggestion
          margin-top: 2.5em;
```

Looking at [the version from this PR](https://table-markup-dot-wptdashboard-staging.uk.r.appspot.com/interop-2023) compared to [the staging environment](https://staging.wpt.fyi/interop-2023), it seems like there are some small visual changes, but this would get partially back to the spacing we were at previously.